docs/developer/typescript/01_gettingStarted/02_install/02_projects.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_label: Sub Projects
+---
+# Installing and running sub projects
+Ethereal Engine can run sub-projects or 'locations'.   
+This pattern is different from what you may be used to because Ethereal Engine runs on the 'outside' and the sub-project is kept in a folder of Ethereal Engine.
+
+Ethereal Engine scans for projects mounted in the /packages/projects/projects sub-folder of Ethereal Engine.
+
+From a bash shell in the Ethereal Engine folder we can install, run and visit a project like so:
+
+```bash
+gh repo clone EtherealEngine/ee-tutorial-basic
+mv ee-tutorial-basic packages/projects/packages/ee-tutorial-basic
+npm install
+npm run dev
+```
+
+Once Ethereal Engine is running, from the web admin panel of Ethereal Engine create a 'location' for the project.   
+See https://localhost:3000/admin.  
```

Comment:
missing /locations here

Reply:
This page probably shouldn't really exist, or should explain projects in a bit more detail than what the other quickstart page is doing.
This specific issue was solved, but its solved in one of the pages earlier. So this page still has this problem because it was how it originally looked.